### PR TITLE
Fix72- Fixed issue with image resize

### PR DESCRIPTION
**Pull request by alhernandez01**
fixed the modal responsive design, and resize

### CODE REVIEW

File: app/components/revisions/damages/show-image-damage/image-damage.show.html [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+        <div class="modal-footer">
+            
+            <div class="modalButton-imageDamage">
+                <button class="btn btn-modalImageDamageCancel" type="button" ng-click="vm.cancel()">Ok</button>
+            </div>
+        </div>
+   
+  
+</div>  
+
```
Comment (johnnyxu-pernix) [inline app/components/revisions/damages/show-image-damage/image-damage.show.html]: Fix indentation.

Comment (johnnyxu-pernix) [inline app/components/revisions/damages/show-image-damage/image-damage.show.html]: Add new line at the end of the file.

**Comment by EscobarMiranda** [comment]
Please rename the PR

**Comment by alhernandez01** [comment]
fixed changes requested.

**Pull request merged**